client/css/style.css

```diff
@@ -1496,6 +1496,8 @@ part/quit messages where we don't load previews (adds a blank line otherwise) */
 
 [contenteditable]:focus {
 	outline: none;
+	-webkit-user-select: text;
+	user-select: text;
```

Comment:
If this is the right fix, you'll want to use the full set of prefixes we use for example [here](https://github.com/thelounge/lounge/blob/master/client/css/style.css#L161-L167):

```css
-webkit-user-select: text;
-moz-user-select: text;
-ms-user-select: text;
user-select: text;
```

While you're at it, could you add `cursor: text;` please? Thanks!

Reply:
I didn't include those because as far as I can tell, only WebKit/Safari uses "none" as its default and the other browsers worked as expected. For certainty's sake, we can do that though.

Reply:
Yeah, let's be consistent, and whenever we tackle #1551, we'll let the auto-prefixer get on with it :)